x/group/types/proposal.go

```diff
@@ -0,0 +1,102 @@
+package types
+
+import (
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+func (p *Proposal) GetMsgs() []sdk.Msg {
```

Comment:
```suggestion
func (p *Proposal) CreateMesages() []sdk.Msg {
```

Reply:
or: MkMessages, MakeMessages...

Reply:
This method is not creating or making messages, it's just getting the already stored messages in the form of `Any`s as `sdk.Msg`s. This is the format we have been using so far in the sdk (see for instance `MsgSubmitEvidence` `GetEvidence()` or `BaseAccount` `GetPubKey()`).